charts/shlink-web/Chart.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: shlink-web
+description: A Helm chart to deploy shlink-web
+icon: https://shlink.io/favicon.svg
+type: application
+version: 1.0.0
```

please start off with a 0.x version

```suggestion
version: 0.1.0
```

We can tag 1.x once these efforts are beyond being an initial release